lib/std/os/linux/tls.zig

```diff
@@ -269,7 +269,7 @@ pub fn setThreadPointer(addr: usize) void {
         },
         .loongarch32, .loongarch64 => {
             asm volatile (
-                \\ mv tp, %[addr]
+                \\ or $tp, $zero, %[addr]
```

Comment:
```suggestion
                \\ move $tp, %[addr]
```

Same output but a bit more readable.

Reply:
https://github.com/loongson/la-asm-manual/blob/main/adoc/CHAPTER-9.Pseudo_Instruction_Set/9.Pseudo_Instruction_Set.adoc

OK, there is no difference to the machine.